lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -334,12 +334,14 @@ def gem_version_compare(first, second)
             Gem::Version.new(first.to_s) <=> Gem::Version.new(second.to_s)
           end
 
-          apipie :method, "Returns content of 'SSL CA file' configured in Settings > Authentication" do
+          apipie :method, "Returns the certifiacte of Foreman Server CA" do
+            desc 'Currently it relies on "SSL CA file" authentication setting, which normally points to the file containing the
+                  CA certificate for Smart Proxies. However in the default deployment, this certificate happens to be the same.'
             example "SSL_CA_CERT=$(mktemp)
                      cat > $SSL_CA_CERT <<CA_CONTENT
                      <%= foreman_server_ca_cert %>
                      CA_CONTENT
-                     curl --cacert $SSL_CA_CERT https://smart-proxy.example.com:8443"
+                     curl --cacert $SSL_CA_CERT https://foreman.example.com:8443"
```

Comment:
Foreman typically doesn't run on 8443. Perhaps you can also use the `foreman_server_url` macro to indicate it's related.

Reply:
ah missed that, I would avoid using that macro (even though it's entirely correct), because it wouldn't be so clear